src/opm/input/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -141,15 +141,15 @@ struct SummaryConfigContext {
          {"PRESSURE", {"FPR" , "RPR" , "BPR"}},
          {"RPV", {"FRPV" , "RRPV" , "BRPV"}},
          {"OIP"  , {"ROIP" , "FOIP" , "FOE"}},
-         {"OIPR" , {"FOIPR"}},
+         {"OIPR" , {"FOIPR", "BOPV"}},
          {"OIPL" , {"ROIPL" ,"FOIPL" }},
          {"OIPG" , {"ROIPG" ,"FOIPG"}},
          {"GIP"  , {"RGIP" , "FGIP"}},
-         {"GIPR" , {"FGIPR"}},
+         {"GIPR" , {"FGIPR", "BGPV"}},
          {"GIPL" , {"RGIPL" , "FGIPL"}},
          {"GIPG" , {"RGIPG", "FGIPG"}},
          {"WIP"  , {"RWIP" , "FWIP"}},
-         {"WIPR" , {"FWIPR"}},
+         {"WIPR" , {"FWIPR", "BWPV"}},
```

Comment:
I don't think you really need this part of the change.  The `required_fields` track which solution variables must be available in order to calculate certain derived summary quantities.  For instance, the `PRESSURE` solution variable must exist in order to calculate the average field pressure (`FPR`), the average region pressure values (`RPR`) or extract pressure values at the block level (`BPR`).

As your OPM-simulators PR (OPM/opm-simulators#3905) proves, you can compute the `BxPV` quantities without any of the `xIPR` quantities in place.  That said, adding these values here doesn't hurt anything but it also doesn't help.